tutorials/02_install.md

```diff
@@ -0,0 +1,75 @@
+\page installation Installation
+
+We recommend following the [Binary Install](#binary-install) instructions to get up and running as quickly and painlessly as possible.
```

Comment:
These anchors don't work on the doxygen-generated tutorials, see here for example:

https://ignitionrobotics.org/api/rendering/4.0/installation.html#binary-install

On `ign-gui` I'm just removing them: https://github.com/ignitionrobotics/ign-gui/pull/127/

Reply:
removed. 4221e97600dddc38c82add1f41e3e1573d949dde